### PR TITLE
Simplify GitHub Action entrypoint (#1909)

### PR DESCRIPTION
**Pull request by simaki**
This PR simplifies `entrypoint.sh` for GitHub Actions by removing duplication of `args` and `black_args` (cf. #1909).

The reason why #1909 uses the input id `black_args` is to avoid an overlap with `args`, but this naming seems redundant.
So let me suggest `option` and `src`, which are consistent with CLI. Backward compatibility is guaranteed; Users can still use `black_args` as well.

Any suggestions for improvements are more than welcome. Thanks!

This PR may not require a changelog entry.

### CODE REVIEW

**Review by cooperlees — APPROVED** [review]
Sorry I missed this. Looks good to me. Lets merge with the deprecated option and remove that in a few months.

My only question, is there a way we can make CI using `black_args` indicate deprecate warnings to the users?

File: action/entrypoint.sh
Comment (simaki) [inline action/entrypoint.sh:4]:
```suggestion
  echo '::warning::WARNING: input `with.black_args` is deprecated. Use `with.options` and `with.src` instead.'
```

**Comment by simaki** [comment]
Yes, we can indicate deprecate warnings to the users: [Setting a warning message - GitHub Actions](https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-a-warning-message).

Please commit the above suggestion to make this change.

**Comment by ichard26** [comment]
If no one objects before then, I'll be merging this in 30ish minutes. Cooperlees already approved and his non-binding comment was also resolved.

**Comment by ichard26** [comment]
Thank you @simaki for your contribution! This should help keep this action maintainable and easy to understand which is always nice since not everyone on the core team is great at this kind of stuff (e.g. me!)

**Review by simaki — COMMENTED** [review]
@ichard26 Hi, could you kindly merge these fixes? I noticed these mistakes by my local run. Sorry about bothering you... Many thanks! (btw continuous testing of GH Actions would be nice but I've got no idea how to do it. seeking idea.)

**Comment by ichard26** [comment]
> ichard26 Hi, could you kindly merge these fixes? I noticed these mistakes by my local run. Sorry about bothering you... Many thanks! (btw continuous testing of GH Actions would be nice but I've got no idea how to do it. seeking idea.)

yeah don't worry about, I'm honestly more surprised I didn't catch that error :disappointed:, here's the hotfix: https://github.com/psf/black/pull/2202

I got some ideas to test out our Action but they will have to wait till later once I get some time.

**Comment by Secret158** [comment]
- [ ] [Missmia][NASA](Forever)Onlygirl

**Pull request merged**
